src/HashBus.Viewer/Program.cs

```diff
@@ -3,12 +3,15 @@
     using System;
     using System.Configuration;
     using System.Linq;
+    using System.Text;
     using System.Threading.Tasks;
 
     class Program
     {
         static Task Main()
         {
+            Console.OutputEncoding = Encoding.UTF8;

```

Comment:
I'd prefer to have this line right before the setting of `Console.Title` so we have all the console set up in one place.